corehq/ex-submodules/casexml/apps/phone/restore.py

```diff
@@ -43,6 +48,9 @@ def add_custom_parameter(key, value):
 # for rapid iteration on fixtures/cases/etc.
 INITIAL_SYNC_CACHE_THRESHOLD = 60  # 1 minute
 
+# Max amount of bytes to have in memory when streaming a file
+MAX_BYTES = 1000000  # 1MB
```

Comment:
Seems reasonable though I don't think we'd have any issue if we increased it to 10Mb which would mean that most responses would be read in a single read.